### DISCUSSION TITLE
第6节课作业(15班)

### DISCUSSION THREAD

**Discussion by ironartisan** [InternLM/Tutorial#96] [作业]
提交第6节课作业

**Comment by bblgbr** [comment]
第六节课作业：https://github.com/bblgbr/play_llm/blob/main/InterLM_Class/06%20OpenCompass%E5%A4%A7%E6%A8%A1%E5%9E%8B%E8%AF%84%E6%B5%8B-%E4%BD%9C%E4%B8%9A.md

**Comment by dqyydq** [comment]
https://blog.csdn.net/qq_52410386/article/details/135759956?spm=1001.2014.3001.5501

**Comment by szhowardhuang** [comment]
基础作业见附件pdf
[OpenCompass介绍.pdf](https://github.com/InternLM/tutorial/files/14020053/OpenCompass.pdf)

**Comment by Mrguanglei** [comment]
https://mqf0lvyk616.feishu.cn/wiki/H3KewsqxWiSTr1kSKXecwvoInIf?from=from_copylink

**Comment by lizhunkg** [comment]
第六课作业： https://note.youdao.com/s/cRtbxHSh

**Comment by sokolo05** [comment]
第六节课作业：https://github.com/sokolo05/Scholar_PuYu/blob/main/01.%E8%AF%BE%E7%A8%8B%E4%BD%9C%E4%B8%9A/%E7%AC%AC6%E8%8A%82_%E4%BD%9C%E4%B8%9A.md

**Comment by ziptoam** [comment]
第六节课作业
【有道云笔记】第六节课笔记与作业
https://note.youdao.com/s/QDJn106x

**Comment by scchy** [comment]
第六节课作业：
https://github.com/scchy/My_Learn/blob/master/LLM_prime/6_InternLM_opc_HW.md

**Comment by woodx9** [comment]
请参考实操相关部分[飞书链接](https://gxloh9mfvxh.feishu.cn/wiki/AhJQw4CTAiTMcWkzK36cC7uenrf?from=from_copylink)

**Comment by IyGnahz** [comment]
笔记&作业 https://g6xuom3zbh.feishu.cn/docx/UcyXd0YUCoMDEKxuW55cvy08nnG

**Comment by yabgdsajghsj** [comment]
https://note.youdao.com/s/ExhlgyS8

**Comment by seienn** [comment]
https://gvxxiiwsufy.feishu.cn/wiki/C5gDwcV57iG8sakTlwKcawYLnIf?from=from_copylink

**Comment by NANDSS1** [comment]
https://www.yuque.com/nandss/gyyg9p/unap8qp23vnxdn6m?singleDoc# 《lesson6 OpenCompass 大模型评测 作业》

**Comment by g29times** [comment]
https://novagen.notion.site/1e2c2725e46e450c956c68e855a098fd?pvs=4
https://colab.research.google.com/drive/1QWn_XWHWXyJJuk8Uq6vm2gP0clvWJIbQ?usp=sharing

**Comment by HarveyLiii** [comment]
使用 OpenCompass 评测 InternLM2-Chat-7B 模型在 C-Eval 数据集上的性能

<img width="2560" alt="截屏2024-02-22 23 20 35" src="https://github.com/InternLM/tutorial/assets/127242206/441327cc-ad3a-4def-9406-eca0124754b2">

<img width="2560" alt="截屏2024-02-22 23 21 31" src="https://github.com/InternLM/tutorial/assets/127242206/64a598fb-a975-4300-86a8-75c8b9e6b60c">
